fctesting/sandbox/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG TMP_BUILD_DIR=/tmp/build
+
+FROM rust:1.32 as build
+
+ARG TMP_BUILD_DIR
+ARG FIRECRACKER_SRC_DIR="/firecracker"
+ARG FIRECRACKER_BUILD_DIR="$FIRECRACKER_SRC_DIR/build"
+ARG CARGO_REGISTRY_DIR="$FIRECRACKER_BUILD_DIR/cargo_registry"
+ARG FIRECRACKER_VERSION="0.15.0"
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        musl-tools
+
+# Add musl to rust std
+RUN mkdir "$TMP_BUILD_DIR" \
```

Comment:
Why do you need to use "$TMP_BUILD_DIR"?  Did you want to make this variable instead of picking a constant path?

Reply:
Naw, it could be a constant path, just thought `ENV` were specifically for environment variables and didn't know what is considered to be a constant other than `ENV`. The `TMP_BUILD_DIR` is used as the test folder where it searches for firecracker, jailer, kernel image, and root files.

Reply:
I would probably just write the literal `/build` instead of using an environment variable.

Reply:
I went with using an `ENV` just due to it being easier to change if we ever do decide to move it from `/build`.